nucleus/wsgi.py

```diff
@@ -15,7 +15,7 @@
 from decouple import config
 
 
-IS_HTTPS = config('HTTPS', default='off') == 'on'
+IS_HTTPS = config('HTTPS', default='off', cast=bool)
```

Comment:
I's about to say that `default='off'` won't work with `cast=bool` but I tested it and it works. +1